llpc/lower/llpcSpirvLowerRayTracing.cpp

```diff
@@ -611,9 +611,9 @@ bool SpirvLowerRayTracing::runImpl(Module &module) {
 void SpirvLowerRayTracing::processLibraryFunction(Function *func) {
   assert(m_shaderStage == ShaderStageCompute);
   auto mangledName = func->getName();
-  const char *traceRayFuncName = m_context->getPipelineContext()->getRayTracingFunctionName(Vkgc::RT_ENTRY_TRACE_RAY);
+  StringRef traceRayFuncName = m_context->getPipelineContext()->getRayTracingFunctionName(Vkgc::RT_ENTRY_TRACE_RAY);
 
-  if (mangledName.startswith(traceRayFuncName)) {
+  if (!traceRayFuncName.empty() && mangledName.startswith(traceRayFuncName)) {
```

Comment:
Should we assert if this name is empty?
The result of compiling an rt pipeline without TraceRay is not really useful, is it?